### ISSUE TITLE
chore: QA Checklist + Script

### ISSUE THREAD

**Issue by dan-jan** [janhq/jan#508]
## Objective

- We need a "QA Checklist" that consists of major user stories
- We will need to run through this QA Checklist on multiple platforms
   - Mac Intel
   - Mac M1/M2/M3
   - Windows
   - Windows on Linux
   - Ubuntu

## Task

- [ ] Capture high-level user "journey" through the app as a QA script
- [ ] Add Script to https://github.com/janhq/jan
- [ ] Implement pre-release QA process + habits

**Comment by 0xSage** [comment]
@imtuyethan following up on this, do you have an initial list?

One idea is to look at the API, and basically every CRUD action is one item that should be on the QA list.

**Comment by 0xSage** [comment]
@imtuyethan happy to pair with you to draft an initial version, based on your new designs. Just add a time to my cal - would like to get this issue unblocked. Critical to get to bug free by 15 dec.

**Comment by 0xSage** [comment]
Updated /specs/product sections to account for this

**Comment by dan-jan** [comment]
Duplicate of #763
